### PR TITLE
(node/lsstcam-vw02.cp.lsst.org) provision host EL9 & add test

### PR DESCRIPTION
**Pull request by badenerb**

### CODE REVIEW

**Review by dtapiacl — DISMISSED** [review]
Please bind this host to a device on IPAM.

File: spec/hosts/nodes/lsstcam-vw02.cp.lsst.org_spec.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'lsstcam-vw02.cp.lsst.org', :sitepp do
+  on_supported_os.each do |os, os_facts|
+    next if os =~ %r{centos-7-x86_64}
```
Comment (jhoblitt) [inline spec/hosts/nodes/lsstcam-vw02.cp.lsst.org_spec.rb]:
```suggestion
    next unless os =~ %r{almalinux-9-x86_64}
```

**Review by jhoblitt — CHANGES_REQUESTED** [review]

**Review by jhoblitt — CHANGES_REQUESTED** [review]
Please squash or improve the commit title of 06abd3d894fb4e7ca7dc7f2a901b403f034df34c

**Review by jhoblitt — APPROVED** [review]
LGTM.

**Comment by badenerb** [comment]
> Please bind this host to a device on IPAM.

Fixed this

**Comment by badenerb** [comment]
> Please bind this host to a device on IPAM.

Fixed

**Pull request merged**